frontend/src/app/profile/logout.tsx

```diff
@@ -44,9 +44,10 @@ export const Logout: React.SFC<{ lang: string }> = ({ lang }) => {
           px: 3,
         }}
       >
-        <Heading mb={2} as="h2">
+        <Heading mb={2} as="h1">
```

Comment:
did you do this to change the font size?

Reply:
not really, more a side effect while I was experimenting with refactoring the code a bit, I want to make everything an h2 and the main one h1.

SEO-wise and dom-wise(? :P) it's best to have the `h1` for `My Profile` as it's the title of the page and `h2` for the others

Reply:
sounds good, just wondering if the font size will come from theme ui or from the UA stylesheet :D

Reply:
in the theme we have the default style of `h1` etc so it will be from theme-ui

Reply:
<img width="313" alt="image" src="https://user-images.githubusercontent.com/3382153/73596969-dfdc0180-4527-11ea-905b-2e749e3b8281.png">